core/src/main/scala/kafka/log/LogSegment.scala

```diff
@@ -68,19 +68,19 @@ class LogSegment private[log] (val log: FileRecords,
   }
 
   def resizeIndexes(size: Int): Unit = {
-    offsetIndex.resize(size)
-    timeIndex.resize(size)
-  }
-
-  def sanityCheck(timeIndexFileNewlyCreated: Boolean): Unit = {
-    if (offsetIndex.file.exists) {
-      offsetIndex.sanityCheck()
-      // Resize the time index file to 0 if it is newly created.
-      if (timeIndexFileNewlyCreated)
-        timeIndex.resize(0)
-      timeIndex.sanityCheck()
-      txnIndex.sanityCheck()
+      offsetIndex.resize(size)
+      timeIndex.resize(size)
     }
+
```

Comment:
Looks like no change here. Can you undo the extra spaces so that this diff is not shown.